products/macos.md

```diff
@@ -25,22 +25,22 @@ releases:
     eol: false
     link: https://support.apple.com/HT213268
     releaseDate: 2022-10-24
-    latestReleaseDate: 2022-12-13
-    latest: '13.1'
+    latestReleaseDate: 2023-01-23
+    latest: '13.2'
 -   releaseCycle: "12"
     codename: "Monterey"
     eol: false
     link: https://support.apple.com/HT212585
     releaseDate: 2021-10-25
-    latestReleaseDate: 2022-12-13
-    latest: '12.6.2'
+    latestReleaseDate: 2023-01-23
+    latest: '12.6.3'
```

Comment:
Latest link should be https://support.apple.com/en-us/HT213604 ?

Reply:
After having a look at the page I think it is just not updated for the time being. Closing this issue.